client/components/shared/nav/NavConfig.tsx

```diff
@@ -20,13 +21,15 @@ export interface NavItem {
 
 export interface MenuSpecificNavItem extends NavItem {
 	isDropDownExclusive?: boolean;
+	isExcludedByFeatureSwitch?: boolean;
 }
 
 interface NavLinks {
 	accountOverview: MenuSpecificNavItem;
 	billing: MenuSpecificNavItem;
 	profile: MenuSpecificNavItem;
 	settings: MenuSpecificNavItem;
+	savedArticles: MenuSpecificNavItem;
```

Comment:
I'm not sure if this is a better solution becuase I think you'd need to conditionally check for the presence of `savedArticles` in other places using the `NavLinks` such as `MMAPageSkeleton` and the new `SavedArticlesComponent`. Given it will be a hopefully temporary exclusion I think it's clearer to have the code doing the work in one file with a explicit nameing to explain what is is doing rather than returning undefined.  Maybe I am thinking about it wrong though? Happy to pair and review.

Reply:
Sure that makes sense. Looks like it's the same approach being taken with the other boolean field too